javascript/javascript3/week3/canvas.css

```diff
@@ -0,0 +1,4 @@
+#canvasId {
```

Comment:
works ! But if you look at your circles there are being bit blurry(low quality) to get the proper resolution you could for example set the actual canvas (context 2d) size to be the full screen with JS

```
 ctx.canvas.width = window.innerWidth; // or screen.width
  ctx.canvas.height = window.innerHeight; // or screen.height
```